Add Multi-Sanbdox Getting Started for STM32F7508-DK (#1090)
MicroEJ/docs: SDK6UserGuide/gettingStarted.rst

   gettingStartedIMXRT1170
   gettingStartedSTM32F7508
   

Multi-Sandbox Evaluation
------------------------

This Getting Started serves as a starting point for evaluating
the Multi-Sandboxing capabilities of the MicroEJ technology.

Run Sandboxed Applications on a ready-to-use Multi-Sandbox Executable
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

MicroEJ provides ready-to-use Multi-Sandbox Executables with basic services
for developing Sandboxed Applications and deploying them easily on your Device.

> This Getting Started explains how to deploy this Multi-Sandbox Executables on a board

Reviewer: "these Multi-Sandbox Executables" or "this Multi-Sandbox Executable"

and run Sandboxed Applications on it.

.. list-table::
   :widths: 50 50
   :header-rows: 0

   * - |logo_st|
     - |image_stm32f7508|
       :ref:`Run Sandboxed Applications on STM32F7508-DK <sdk_6_getting_started_stm32f7508_multisandbox>`

.. _sdk6_getting_started_kernel_green:

Build your own Multi-Sandbox Executable
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

This Getting Started is a step-by-step guide explaining how to build a Multi-Sandbox Executable.

> This Executable will be built from the `Kernel GREEN <https://github.com/MicroEJ/Kernel-GREEN>`__ sources.

Reviewer: ```suggestion
This Executable will be built from `Kernel GREEN <https://github.com/MicroEJ/Kernel-GREEN>`__ sources.
```

Author: Fixed

Kernel GREEN provides a turnkey Kernel code providing basic services 

> for developing a Sandboxed Application and deploying them easily on your Device.

Reviewer: ```suggestion
to develop a Sandboxed Application and deploy it easily on your Device.
```

Author: Fixed
